submariner/cable-driver-protocols-enhancement.md

```diff
@@ -0,0 +1,189 @@
+# Cable Driver Protocol Support Enhancements
+
+## Summary
+
+Submariner today supports three cable drivers: VXLAN, IPSec (tunnel mode) and WireGuard. The
+goal of this proposal is to outline additional protocols and enhancements for the Submariner
+Cable Driver. These include the additional support of IP-in-IP for unencrypted connections
+between clusters and the support of IPSec transport mode.
+
+For IP in IP, the benefits fall into the following categories:
+
+* **Performance**: IP in IP is has a lower overhead than VXLAN which could lead to performance
+improvements in terms of packet overhead (bandwidth consumption) as well as the cost of processing
+packets. Some of the performance improvements are shown below [1]:
+
+  * ![Throughput rate vs Kernel](./images/tx-tput-rate.png)
+  * ![Throughput rate Gbps](./images/tx-tput-gbps.png)
+  * ![CPU cycles](./images/cpu-cycles.png)
+
+* **Interoperability**:IP in IP can be used to transport packets between domains when the protocol
+in those domains (e.g. IPv6) is not supported by intermediary networks (e.g. IPv4).
+
+For Overlay-over-IPSec in transport mode with further configuration, the benefits fall into the
+following categories:
+
+* **Performance**: Taking advantage of and overlay protocol wrapped in offloaded IPSec in transport mode can
```

Comment:
```suggestion
* **Performance**: Taking advantage of an overlay protocol wrapped in offloaded IPSec in transport mode can
```

Reply:
I've fixed this up in https://github.com/submariner-io/enhancements/pull/103